src/styles/blocks/card.css

```diff
@@ -0,0 +1,85 @@
+.card {
+  display: inline-flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
```

Comment:
This is default value.

Reply:
Please answer what it means. Should I relocate this value or put it in another file/folder?